.about.yml

```diff
@@ -2,63 +2,112 @@
 
 # What kind of team owns the repository? (required)
 # values: guild, working-group, project
-owner_type:
+owner_type: project
 
 # What is your project's current status? (required)
 # values: discovery, alpha, beta, live
-stage:
+stage: alpha
 
 # Should this repo have automated tests? If so, set to `true`. (required)
 # values: true, false
-testable:
+testable: true
 
 # What are the licenses that apply to the project and/or its components? (required)
 # Items by property name pattern:
 #   .*:
 #     name: Name of the license from the Software Package Data Exchange (SPDX): https://spdx.org/licenses/
 #     url: URL for the text of the license
 licenses:
-  placeholder_label:
+  codeclimate-test-reporter:
+    name: MIT
+    url: https://github.com/codeclimate/python-test-reporter/blob/master/LICENSE.txt
+  coverage:
+    name: Apache-2.0
+    url: https://bitbucket.org/ned/coveragepy/src/3ede4137fe6ddf7bd7dc1427055c9fb44f7da5ad/LICENSE.txt?at=default&fileviewer=file-view-default
+  dj-database-url:
+    name: BSD-2-Clause
+    url: https://github.com/kennethreitz/dj-database-url/blob/master/LICENSE
+  Django:
+    name: BSD-3-Clause
+    url: https://github.com/django/django/blob/master/LICENSE
+  django-filter:
+    name: BSD-3-Clause
+    url: https://github.com/carltongibson/django-filter/blob/develop/LICENSE
+  djangorestframework:
+    name: BSD-2-Clause
+    url: https://github.com/tomchristie/django-rest-framework/blob/master/LICENSE.md
+  djangorestframework-csv:
+    name: BSD-2-Clause
+    url: https://github.com/mjumbewu/django-rest-framework-csv#license
+  factory_boy:
+    name: MIT
+    url: https://github.com/FactoryBoy/factory_boy/blob/master/LICENSE
+  httmock:
+    name: Apache-2.0
+    url: https://github.com/patrys/httmock/blob/master/LICENSE
+  markdown2:
+    name: MIT
+    url: https://github.com/trentm/python-markdown2/blob/master/LICENSE.txt
+  Pillow:
+    name: PIL (MIT)
+    url: https://github.com/python-pillow/Pillow/blob/master/LICENSE
+  psycopg2:
+    name: LGPL-3.0
+    url: https://github.com/psycopg/psycopg2/blob/master/LICENSE
+  pycodestyle:
+    name: MIT
+    url: https://github.com/PyCQA/pycodestyle/blob/master/LICENSE
+  PyJWT:
+    name: MIT
+    url: https://github.com/jpadilla/pyjwt/blob/master/LICENSE
+  requests:
+    name: Apache-2.0
+    url: https://github.com/kennethreitz/requests/blob/master/LICENSE
+  whitenoise:
+    name: MIT
+    url: https://github.com/evansd/whitenoise/blob/master/LICENSE
 
 # Who is the partner for your project? (Use the full name of the partner documented here: https://github.com/18F/dashboard/blob/staging/_data/partners.yml)
 partners:
-- 
+-
 
 # The main point of contact(s) and/or the issue reporting system for your project, and either a `mailto:` link or URL for each contact.
 # Items:
 # - url: URL for the link
 #   text: Anchor text for the link
 contact:
```

Comment:
That's not a required field for `.about.yml`, but it doesn't hurt to fill it in. To my knowledge, the only thing that we use `.about.yml` for is the dependency license documentation.